osu.Game/Skinning/DefaultSkin.cs

```diff
@@ -22,6 +23,29 @@ namespace osu.Game.Skinning
 {
     public class DefaultSkin : Skin
     {
+        private readonly IStorageResourceProvider resources;
+
+        private static readonly IReadOnlyDictionary<GameplaySkinSamples, string> sample_mapping
+            = new Dictionary<GameplaySkinSamples, string>
+            {
+                { GameplaySkinSamples.ResultScoreTick, @"Results/score-tick" },
+                { GameplaySkinSamples.ResultBadgeTick, @"Results/badge-dink" },
+                { GameplaySkinSamples.ResultBadgeTickMax, @"Results/badge-dink-max" },
+                { GameplaySkinSamples.ResultSwooshUp, @"Results/swoosh-up" },
+                { GameplaySkinSamples.ResultRank_D, @"Results/rank-impact-fail-d" },
+                { GameplaySkinSamples.ResultRank_B, @"Results/rank-impact-fail" },
+                { GameplaySkinSamples.ResultRank_C, @"Results/rank-impact-fail" },
+                { GameplaySkinSamples.ResultRank_A, @"Results/rank-impact-pass" },
+                { GameplaySkinSamples.ResultRank_S, @"Results/rank-impact-pass" },
+                { GameplaySkinSamples.ResultRank_SS, @"Results/rank-impact-pass-ss" },
+                { GameplaySkinSamples.ResultApplause_D, @"Results/applause-d" },
+                { GameplaySkinSamples.ResultApplause_B, @"Results/applause-b" },
+                { GameplaySkinSamples.ResultApplause_C, @"Results/applause-c" },
+                { GameplaySkinSamples.ResultApplause_A, @"Results/applause-a" },
+                { GameplaySkinSamples.ResultApplause_S, @"Results/applause-s" },
+                { GameplaySkinSamples.ResultApplause_SS, @"Results/applause-s" }
+            };
```

Comment:
Shouldn't exist. I think these are fine to just be inside a standard `SampleInfo`. We can consider skinning support at a much later date.